lua/ulx/modules/sh/cfc_curse.lua

```diff
@@ -72,6 +161,26 @@ function cmd.cursePlayers( callingPlayer, targetPlayers, effectName, durationMin
         return
     end
 
+    if smallestAmount == largestAmount and smallestAmount == 1 then
+        amount = false
+    end
+
+    if amount then
+        local amountStr
+
+        if smallestAmount == largestAmount then
+            amountStr = tostring( smallestAmount )
+        else
+            amountStr = smallestAmount .. " to " .. largestAmount
+        end
+
+        local durationAppend, briefly = getDurationStrings( durationSeconds, false )
+
+        ulx.fancyLogAdmin( callingPlayer, isSilent, "#A" .. briefly .. " cursed #T with " .. amountStr .. " random effects" .. durationAppend, targetPlayers )
```

Comment:
Thoughts? "Effects" seems a little boring
```suggestion
        ulx.fancyLogAdmin( callingPlayer, isSilent, "#A" .. briefly .. " hexed #T with " .. amountStr .. " random curses" .. durationAppend, targetPlayers )
```